services/ops/LogStatisticsAgent/README.md

```diff
@@ -0,0 +1,38 @@
+## Log Statistics Agent
+
+The Log Statistics agent periodically reads the "volttron.log" file, computes the size delta from the previous hour 
+and publishes the difference with a timestamp.  It also publishes standard deviation of the size delta every 24 hours.
+This agent is useful for ___.
+
+
+### Configuration
+
+The Log Statistics agent has 4 configuration values:
+
```

Comment:
are these all required? should say so or mark which are and aren't
Again I like using line code formate for `variable_names` but that's personal style preference.